openpype/lib/applications.py

```diff
@@ -973,6 +973,9 @@ def __init__(
 
         self.env_group = env_group
 
+        stdout = data.pop("stdout", None)
```

Comment:
This file can be reverted to previous state, currently it is confusing to pop stdout/stderr and then reasign it back.

I though I already wrote it in a comment, not sure if it was this PR?

Reply:
If I revert to original state, then I dont get any stdout from the launched app here; https://github.com/ynput/OpenPype/blob/4a639b41efaa00680abdf77d0b8616283929bc4a/tests/lib/testing_classes.py#L353
Have you got a suggestion to modify the stdout and stderr for launching the app?

Reply:
Ok, then please do this, the stdout/stderr handling is scattered and it's hard to spot the goal.
```python
        stdout = self.data.pop("stdout", None)
        stderr = self.data.pop("stdout", None)
        # Force to use DEVNULL if 'sys.stdout' is 'None'
        # - this is because of UI build on Windows
        if not sys.stdout:
            stdout = subprocess.DEVNULL
            stderr = subprocess.DEVNULL

        if stdout is not None:
            self.kwargs["stdout"] = stdout
        if stderr is not None:
            self.kwargs["stderr"] = stderr
```

Reply:
done